tests/acceptance/features/apiGraph/editGroup.feature

```diff
@@ -23,6 +23,7 @@ Feature: edit group name
       | grp1      | $x<=>[y*z^2]! |
       | grp1      | staff?group   |
       | grp1      | 50%pass       |
+      | grp1=     | grp1          |
```

Comment:
I added this cases because = fails because of https://github.com/owncloud/ocis/issues/5968 and we'll know when it is fixed